api/base/serializers.py

```diff
@@ -227,14 +239,18 @@ def resolve_url(self, obj):
         # Presumably, if you have are expecting a value but the value is empty, then the link is invalid.
         for item in kwarg_values:
             if kwarg_values[item] is None:
-                return None
-        return utils.absolute_reverse(
+                raise SkipField
+        url = utils.absolute_reverse(
             self.endpoint,
             args=arg_values,
             kwargs=kwarg_values,
             query_kwargs=query_kwarg_values,
             **self.reverse_kwargs
         )
+        if not url:
```

Comment:
Under what circumstances does `absolute_reverse` return a falsy value? 

This check may be unnecessary, unless there's a case where a URL would fail to build without raising an exception.